cli/lib/csv.ts

```diff
@@ -6,21 +6,32 @@ import { Stream } from 'rdf-js'
 import * as Csvw from '@rdfine/csvw'
 import { Hydra } from 'alcaeus/node'
 import { readable } from 'duplex-to'
+import { CsvSource } from '@cube-creator/model'
+import fetch from 'node-fetch'
 
 export function openFromCsvw(csvw: Csvw.Table) {
   const csvStream = new PassThrough()
 
   Promise.resolve().then(async () => {
-    const { response } = await Hydra.loadResource(csvw.url!, {
-      accept: 'text/csv',
-    })
+    const { response, representation } = await Hydra.loadResource<CsvSource>(csvw.url!)
+    if (!representation?.root) {
+      csvStream.emit('error', new Error('Failed to load Source. Response was: ' + response?.xhr.statusText))
+      return
+    }
+
+    if (!representation.root.associatedMedia.contentUrl) {
+      csvStream.emit('error', new Error('Failed to load CSV. Missing link from source'))
+      return
+    }
+
+    const csvResponse = await fetch(representation.root.associatedMedia.contentUrl.value)
```

Comment:
For some reason which I cannot understand minio fails to serve the file when fetched using alcaeus 🤷